create-packages.sh

```diff
@@ -18,14 +18,15 @@ fi
 appname=myapp
 version=1.0.0-0
 
-pack_flags=''
-if [[ $(tarantool -V) == *"Target: Darwin"* ]]; then
-    pack_flags+=' --use-docker'
+pack_flags='--use-docker'
 
-    if [[ $(tarantool -V) == "Tarantool Enterprise"* && -z "${TARANTOOL_SDK_PATH}" ]]; then
+if [[ $(tarantool -V) == "Tarantool Enterprise"* && -z "${TARANTOOL_SDK_PATH}" ]]; then
+    if [[ $(tarantool -V) == *"Target: Darwin"* ]]; then
         echo "Set the path to Linux Tarantool SDK using the TARANTOOL_SDK_PATH environment variable!"
         exit 1
     fi
+
+    pack_flags+=' --sdk-local'
```

Comment:
For what this flag? Why was it not there before?

Reply:
Fixed!